devel/changes

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""
+Extract changelog for a specified version (or __NEXT__ if no version
+specified).
+"""
+from pathlib import Path
+from sys import argv, exit, stdout
+
+
+CHANGELOG = Path(__file__).parent / "../CHANGES.md"
+
+
+def main(version = "__NEXT__"):
+    with CHANGELOG.open(encoding = "utf-8") as file:
+        for line in file:
+            # Find the heading for this version
+            if line.startswith(f"# {version}") or line.startswith(f"# {version} "):
```

Comment:
_(non-blocking)_

If `line.startswith(f"# {version}")` is true then `line.startswith(f"# {version} ")` will also be true, so the latter is not needed?

```suggestion
            if line.startswith(f"# {version}"):
```

Reply:
Ah, I got this wrong. Thanks for spotting it. It should be.

```suggestion
             if line == f"# {version}" or line.startswith(f"# {version} "): 
```

(The goal is to avoid matching `# 1.2.10` when the version is `1.2.1` and account for `# __NEXT__` not having a trailing space.)

Reply:
Fixed.